addenda11_test.go

```diff
@@ -70,14 +70,10 @@ func BenchmarkAddenda11Parse(b *testing.B) {
 func testAddenda11ValidRecordType(t testing.TB) {
 	addenda11 := mockAddenda11()
 	addenda11.recordType = "63"
-	if err := addenda11.Validate(); err != nil {
-		if e, ok := err.(*FieldError); ok {
-			if e.FieldName != "recordType" {
-				t.Errorf("%T: %s", err, err)
-			}
-		} else {
-			t.Errorf("%T: %s", err, err)
-		}
+	err := addenda11.Validate()
+	// TODO: are we not expecting any errors here?
+	if !base.Match(err, nil) {
```

Comment:
return the fieldError	

if addenda11.recordType != "7" {
		fieldError("recordType", NewErrRecordType(7), addenda11.recordType)
	}